src/aosm/azext_aosm/tests/latest/test_nsd.py

```diff
@@ -6,16 +6,149 @@
 import os
 from dataclasses import dataclass
 import json
+import shutil
+import subprocess
 from pathlib import Path
 from unittest.mock import patch
 from tempfile import TemporaryDirectory
-from typing import Any, Dict
+
+import jsonschema
 
 from azext_aosm.custom import generate_design_config, build_design
 
```

Comment:
I should have asked in the unit test review, but why aren't we using PyTest?

Reply:
I think the basic examples for the CLI just used unit test, I was worried that `azdev test` would be incompatible with them.  But it looks like other extensions do use pytest, so we could move over.  I'm not going to do that right now though.